torch/testing/_internal/common_utils.py

```diff
@@ -616,9 +616,10 @@ def to_gpu(obj, type_map=None):
         type_map = {}
     if isinstance(obj, torch.Tensor):
         assert obj.is_leaf
-        t = type_map.get(obj.type(), get_gpu_type(obj.type()))
```

Comment:
`get_gpu_type` is only used at one other place, so it would be awesome if you could update that too!
https://github.com/pytorch/pytorch/blob/master/test/test_cuda.py#L770

Reply:
also nit - let's just change this to `t = type_map.get(obj.type(), obj.type())` and change the line below to `res = obj.clone().type(t).cuda()`

Reply:
yeah right makes sense! let's do that